src/routes/Header.svelte

```diff
@@ -0,0 +1,117 @@
+<script>
+	import '$lib/css/main.css';
+	import Navbar from './Navbar.svelte';
```

Comment:
Ideally we should colocate both Header and Navbar into like, `$lib/common` so that they're easy to import from anywhere (e.g. `import Header from '$lib/common/Header.svelte'`)

Reply:
Will we really import it from other places? Right now, I put the components that are only being used once in the route directory, like the [sverdle](https://github.com/sveltejs/kit/tree/master/packages/create-svelte/templates/default/src/routes/sverdle) template. Later branches also do this. But we probably only have a few components that are reused in more than one place. Or do you mean only the components used in the layout should be moved to the `lib` directory?

Reply:
It's not super pressing, I guess with how SvelteKit already reserves `+`-prefixed files for actual routes, this is also a decent approach. Will merge as-is then.